org.scala-ide.sdt.debug/plugin.xml

```diff
@@ -142,17 +142,20 @@
          point="org.eclipse.ui.menus">
       <menuContribution
             allPopups="false"
-            locationURI="toolbar:org.scala-ide.sdt.debug.asyncView">
-         <command
-               commandId="org.scala-ide.sdt.debug.stepMessageOut"
-               icon="icons/bang.png"
-               label="Step Message Out"
-               style="push"
-               tooltip="Step until a message is received">
-            <visibleWhen
-                  checkEnabled="true">
-            </visibleWhen>
-         </command>
+            locationURI="toolbar:org.eclipse.ui.main.toolbar">
```

Comment:
I try to find it in eclipse doc. As I said it cannot be in Async View which
shows history not current state

2015-09-03 12:22 GMT+02:00 Simon Schäfer <notifications@github.com>:

> In org.scala-ide.sdt.debug/plugin.xml
> <https://github.com/scala-ide/scala-ide/pull/1000#discussion_r38631403>:
>
> > @@ -142,17 +142,20 @@
> >           point="org.eclipse.ui.menus">
> >        <menuContribution
> >              allPopups="false"
> > -            locationURI="toolbar:org.scala-ide.sdt.debug.asyncView">
> > -         <command
> > -               commandId="org.scala-ide.sdt.debug.stepMessageOut"
> > -               icon="icons/bang.png"
> > -               label="Step Message Out"
> > -               style="push"
> > -               tooltip="Step until a message is received">
> > -            <visibleWhen
> > -                  checkEnabled="true">
> > -            </visibleWhen>
> > -         </command>
> > +            locationURI="toolbar:org.eclipse.ui.main.toolbar">
>
> Changing the locationURI is not correct. We want to have the icon in the
> Async Stack view (or in the Debug view) but not in th emain menu toolbar.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scala-ide/scala-ide/pull/1000/files#r38631403>.
>